doc/labels/sanitize.md

```diff
@@ -0,0 +1,29 @@
+---
+name: sanitize
+rules:
+  - deserialization-of-untrusted-data
+---
+
+Ensures that data is safe and trusted, transforming it if necessary, and returning `falsey` or
+raising an exception if it's impossible to make the data safe.
+
+A function with this label can be used to convert untrusted data such as direct user input or HTTP
+request parameters into trusted data.
+
+Note that this is not the same as ensuring that a parameter satisfies business logic constraints -
+such as presence or max length. It's a security check that ensures the data cannot cause downstream
+harm.
+
+To be considered successful, a `sanitize` function must return a `truthy` value.
```

Comment:
This is the same doc that you have already corrected in the main branch. Do you need to rebase this?

Reply:
Thanks, I got confused.